apps/framework-cli/src/project.rs

```diff
@@ -36,6 +38,19 @@ use crate::project::typescript_project::TypescriptProject;
 use crate::utilities::constants::{APP_DIR, APP_DIR_LAYOUT, CLI_PROJECT_INTERNAL_DIR, SCHEMAS_DIR};
 use crate::utilities::constants::{FLOWS_DIR, PROJECT_CONFIG_FILE};
 
+lazy_static! {
+    pub static ref PROJECT: Arc<Mutex<Project>> = Arc::new(Mutex::new(Project {
```

Comment:
We don't need `Arc` here because it'll live forever as a global variable.

If we can initialize it (from config files and env vars) we won't need mutability, and we can drop the `Mutex` as well.

Reply:
@phiSgr please see the discussion on the Linear ticket: https://linear.app/514/issue/514-601/refactor-project-to-use-lazy-static